### ISSUE TITLE
This Is the Real Reason You Don’t Have Time for the Things You Love

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#59]
# TL;DR
We can change and fix our time and priorities to work on things we enjoy/important

# Key Takeaways

- **It’s not your motivation. It’s your approach.** Highly productive people who consistently reach their goals don’t find the time for the things they love; they make it. And believe it or not, you already know how that’s done. You do it every time you squeeze something like a doctor’s appointment or a visit to the DMV into your schedule despite being just as busy as anyone else.

- **We always have time for things that are priorities.** Try rephrasing the things you tell yourself as to why you’re not actively working on your passions, and see how they sound. It’s a lot tougher to say “my health isn’t a priority for me right now” instead of “I don’t have time to work out and eat right”, isn’t it. Or to admit that maybe you’re not working on your art because creating isn’t currently the priority it should be.

- **Start small, but be consistent.** You don’t need to free up hours of time to make progress toward a goal. Smaller chunks of time can be just as effective when training yourself to prioritize your passions, and they add up. Try 10 minutes to start with. It’s long enough to make a difference, but short enough that you’ll be able to stick with it.

- **Really look at how you spend your time, and reclaim what you waste.** Over the next couple of days, take a look at how you really spend your time and identify activities that don’t actually bring anything to your life. Consider either cutting those things out of the equation altogether or coming up with ways to make them more enriching and less of a time suck. Spend the time you save pursuing your goals, developing your skills, and actively focusing on your passions instead. Not only will you make amazing progress, but you’ll love how much more productive you feel, especially once you start seeing the results.

- **Make random pockets of “wait time” more productive.** The next time you find yourself in a holding pattern waiting for someone or something else, don’t reach for your phone and start scrolling the way you normally would. Do something small that pushes you closer to really accomplishing something instead.

- **Get comfortable with saying “no”.** People are conditioned to believe a nice person, a good friend, or a loving family member never says “no” when asked for their time or the use of their resources. It’s time to dismantle that thinking. You have the same 24 hours each day that everyone else has, so saying “yes” to one thing means saying “no” to something else. Make sure you’re saying “yes” to what’s really important to you.

- **Time is precious and limited.** Do the things that really matter to you, and do them now. Make ample time for them by getting your priorities straight and keeping them that way. Even the small decisions you make about how to manage your time every day count, so choose wisely.

# Link
“This Is the Real Reason You Don’t Have Time for the Things You Love” by Shannon Hilson https://link.medium.com/DsFVaYxvP5
